framework/src/media/voice/SoftwareEndPointDetector.cpp

```diff
@@ -50,26 +50,5 @@ bool SoftwareEndPointDetector::startEndPointDetect(uint32_t timeout)
 	return true;
 }
 
-bool SoftwareEndPointDetector::processEPDFrame(short *sample, int numSample)
```

Comment:
RecognitionManager calling that function directly, please check it.
We should consider below.
1. sw based epd(check the result of epd for every recording - silence check)
2. hw based epd (silence check based on h/w)
3. hw based epd (callback mechanism)

Reply:
I'll keep in mind, but do it another PR